mobile_verifier/src/settings.rs

```diff
@@ -69,7 +60,9 @@ impl Settings {
             .and_then(|config| config.try_deserialize())
     }
 
-    pub fn verification_offset_duration(&self) -> Duration {
-        Duration::minutes(self.verification_offset_minutes)
+    pub fn start_after(&self) -> DateTime<Utc> {
```

Comment:
this setting sets the FileInfoPoller look back behavior, so it will use this value if there are no records in the `files_processed` table to tell it where to start from.